challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map {|a| a.capitalize}
```

Comment:
Rubyists tend to write this as: `input.map(&:capitalize)`. Google "ruby symbol to_proc" for more info. It's hard to explain.